doc/querying.rdoc

```diff
@@ -715,6 +715,20 @@ aggregation:
 
   Album.select_group(:artist_id).select_append{sum(num_tracks).as(tracks)}
   # SELECT artist_id, sum(num_tracks) AS tracks FROM albums GROUP BY artist_id
+  
+You can combine select_group, select_append while we are using a join and a where statement:
+
+  Album.
+  left_join(:artists.
```

Comment:
Since you are left joining to a subselect, I'm guessing this should be `DB[:artists]` or `Artist`.